src/locales/fr.json

```diff
@@ -161,7 +160,7 @@
     "tech": "Tech",
     "clouds": "Clouds & coffres",
     "finance": "Emploi & finance",
-    "shortcuts": "Raccourcis"
+    "shortcuts": "App supplémentaires"
```

Comment:
App**s** ?

Reply:
Good question 🤔 I'm going to stick with this as it's the working I've been given